CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## Changelog
+
+### Version 1.0.0
```

Comment:
oh wait, `1.0.0` is set to be the next version or typo?

Reply:
Yes, next version should be `1.0.0`.